### PR TITLE
livescript changes

### PR DESCRIPTION
**Pull request by Announcement**
- `lib/tilt.rb` is what i'd like to push.
- `lib/tilt/livescript.rb` is up for discussion, i'm indifferent either way.
- `tilt.gemspec` please don't accept this file.

### CODE REVIEW

File: lib/tilt.rb [hunk 1]
```diff
@@ -140,7 +140,7 @@ def clear
   register_lazy :HamlTemplate,         'tilt/haml',      'haml'
   register_lazy :LessTemplate,         'tilt/less',      'less'
   register_lazy :LiquidTemplate,       'tilt/liquid',    'liquid'
-  register_lazy :LiveScriptTemplate,   'tilt/livescript','ls'
+  register_lazy :LiveScriptTemplate,   'tilt/livescript','ls', 'livescript'
```
Comment (judofyr) [inline lib/tilt.rb]: This is all good by me

File: lib/tilt/livescript.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ def prepare
     end
 
     def evaluate(scope, locals, &block)
-      @output ||= LiveScript.compile(data, options)
+      @output ||= LiveScript.compile(data, {bare: true, header: false})
```
Comment (judofyr) [inline lib/tilt/livescript.rb]: This should at least be `{bare: true, header: false}.merge(options)` so you can override the options, but in general I'm not a fan of adding Tilt-defaults. The LiveScript library already has its own defaults.

File: lib/tilt.rb (hunk 1 above)
Comment (Announcement) [reply lib/tilt.rb]: thank you.

File: lib/tilt/livescript.rb (hunk 2 above)
Comment (Announcement) [reply lib/tilt/livescript.rb]: yeah i was thinking the same tbh

**Comment by jeremyevans** [comment]
It looks like the only change we may want to accept here is adding support for the `livescript` extension.

**Pull request closed without merging**